python/ray/experimental/internal_kv.py

```diff
@@ -1,26 +1,25 @@
+import ray
 from typing import List, Union
 
-import ray
 from ray._private.client_mode_hook import client_mode_hook
 
 
 @client_mode_hook
-def _internal_kv_initialized():
+def _internal_kv_initialized(target=None):
```

Comment:
Also, this `target` argument doesn't appear to be used.

Reply:
Good catch! I forgot to remove this when fixing lifecycle issues!